docs/security/host-certs.md

```diff
@@ -1,45 +1,34 @@
-a service name to the hostname (e.g. http/mymachine.mydomain.edu for an Apache
-httpd service certificate).
-
-Since October 2015 the OSG has run its own OSG CA service that handled host and service certificate requests.
-At the end of May 2018, this service will be retired;
-see [the policy details here](https://opensciencegrid.org/technology/policy/service-migrations-spring-2018/#osg-ca).
-To replace the OSG CA service, we suggest using one or more of the following services depending on your site's needs:
+!!! note
+    This document describes how to get **host** certificates.
+    For instructions on how to get **user** certificates, see the [User Certificates document](user-certs.md).
+
+Host certificates are [X.509 certificates](https://en.wikipedia.org/wiki/X.509) that are used to securely identify
+servers and to establish encrypted connections between services and clients.
+In the OSG, some grid resources (e.g., HTCondor-CE, XRootD, GridFTP) require host certificates.
+If you are unsure if your host needs a host certificate, please consult the installation instructions for the software
+you are interested in installing.
+
+To acquire a host certificate, you must submit a request to a Certificate Authority (CA).
+We recommend requesting host certificates from one of the following CA services:
 
 - [InCommon](https://www.incommon.org/cert/): an IGTF-accredited CA for services that interact with the WLCG;
   requires a subscription, generally held by an institution
 - [Let's Encrypt](https://letsencrypt.org/): a free, automated, and open CA frequently used for web services;
   see the [security team's position on Let's Encrypt](https://opensciencegrid.org/security/LetsEncryptOSGCAbundle/)
   for more details
+- If neither of the above options work for your site, the OSG also accepts all
+  [IGTF-accredited CAs](http://repo.opensciencegrid.org/pacman/cadist/).
 
-!!! tip "Recommendation"
-    Certificates issued from the OSG CA before the service retirement will still be valid until they expire.
-    Therefore, we recommend that you request certificates for any currently existing or planned hosts and services
-    prior to retirement.
-
-After reading this document you should be able to apply for and install a host
-or service certificate on a grid resource.  This document does not explain how
-to apply for a grid user certificate. To learn how to apply for a grid user
-certificate click [here](user-certs.md) instead!
+Use this page to learn how to request and install host certificates on an OSG resource.
 
 Before Starting
 ---------------
 
-Before requesting a new host or service certificate, you should use **openssl** to 
-check if you have a valid certificate already. If so, you may
-safely skip this document:
+Before requesting a new host certificate, use `openssl` to check if your host already has a valid certificate.
+If so, you may safely skip this document:
```

Comment:
You should tell them to look at the dates to verify that they include the present.